packages/console/src/pages/ConnectorDetails/components/DeleteConnectorConfirmModal/index.tsx

```diff
@@ -0,0 +1,45 @@
+import type { ConnectorResponse } from '@logto/schemas';
+import { ConnectorType } from '@logto/schemas';
+import { Trans, useTranslation } from 'react-i18next';
+
+import ConfirmModal from '@/components/ConfirmModal';
+import UnnamedTrans from '@/components/UnnamedTrans';
+
+type Props = {
+  data: ConnectorResponse;
+  isOpen: boolean;
+  onCancel: () => void;
+  onConfirm: () => void;
+};
+
+const DeleteConnectorConfirmModal = ({ data, isOpen, onCancel, onConfirm }: Props) => {
+  const { t } = useTranslation(undefined, { keyPrefix: 'admin_console' });
+  const isSocial = data.type === ConnectorType.Social;
+
+  return (
+    <ConfirmModal
```

Comment:
nit: I have put a global shared ConfirmModal provider.  simply use `useConfirmModal`. No need to build a separate confirm model everywhere.  Not a blocker, please create an issue to optimize later.

Reply:
LOG-5792